roles/edpm_reboot/defaults/main.yaml

```diff
@@ -17,7 +17,8 @@
 
 # All variables intended for modification should be placed in this file.
 
-edpm_reboot_force_reboot: false
-edpm_reboot_nova_compute_config_dir: /var/lib/openstack/config/nova
```

Comment:
im not sure why this was ever using /var/lib/openstack/config/nova

that was obviously a bug because that directory si reserved for use by the edpm_nova role

we will delete anything in that and recreated every time it runs.

so +1 but we may want to add a linint tests for this at some point to ensure other roles to try to modify files in /var/lib/openstack/config/nova

Reply:
well, in first iteration we took care of computes only. Later it was brought to attention that there are other node types we should check and not reboot.